CONTRIBUTING.md

```diff
@@ -164,6 +164,13 @@ For example, if the latest tag is 1.1.3:
 $> git clone https://github.com/apple/swift-crypto --branch 1.1.3
 ```
 
+5. Clone [swift-collections](https://github.com/apple/swift-collections) beside the SwiftPM directory and check out tag with the [latest version](https://github.com/apple/swift-collections/tags).
+
+For example, if the latest tag is 0.0.3:
+```bash
+$> git clone https://github.com/apple/swift-collections --branch 0.0.3
+```
+
```

Comment:
Could the command examples be indented by 3 spaces so it's structurally more clear that they're part of the list items?

Also, could argument parser and swift crypto's versions in the commands be updated to 0.4.3 and 1.1.4, to align with what's in `Package.swift`?